api/producer/countDocumentReference/src/v1/handler.py

```diff
@@ -0,0 +1,71 @@
+from enum import Enum
+from logging import Logger
+from typing import Any
+
+from aws_lambda_powertools.utilities.parser.models import APIGatewayProxyEventModel
+from lambda_pipeline.types import FrozenDict, LambdaContext, PipelineData
+from lambda_utils.logging import log_action
+
+from nrlf.core.common_steps import parse_headers
+from nrlf.core.constants import DbPrefix
+from nrlf.core.errors import assert_no_extra_params
+from nrlf.core.model import PaginatedResponse, ProducerRequestParams, key
+from nrlf.core.repository import Repository, type_filter
+from nrlf.core.transform import create_bundle_count
+from nrlf.core.validators import validate_type_system
+from nrlf.producer.fhir.r4.model import NextPageToken, RequestQuerySubject
+
+
+class LogReference(Enum):
+    COUNT_001 = "Searching for document references"
```

Comment:
shouldn't this just be `COUNT` rather than `FOO_001`?

Reply:
Yes